handlers/practitioner_resource.go

```diff
@@ -166,3 +166,117 @@ func HandleDeletePractitioner(svc dao.Service) http.Handler {
 		w.WriteHeader(statusCode)
 	})
 }
+
+// HandleAppointPractitioner adds appointment details to a practitioner resource on a transaction
+func HandleAppointPractitioner(svc dao.Service) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+		// Check for a transaction id in request
+		vars := mux.Vars(req)
+		transactionID := utils.GetTransactionIDFromVars(vars)
+		if transactionID == "" {
+			log.ErrorR(req, fmt.Errorf("there is no Transaction ID in the URL path"))
+			m := models.NewMessageResponse("Transaction ID is not in the URL path")
```

Comment:
Message response shouldn't start with a capital, probably a good idea to keep it in line with the format on line 140